code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,295 @@
+//ripped off of wall gazer, sorry devs
+/mob/living/simple_animal/hostile/abnormality/mailpile
+	name = "Letters on Standby"
+	desc = "A pile of stamped letters, none reaching their intended receiver."
+	icon = 'ModularTegustation/Teguicons/32x48.dmi'
+	icon_state = "mailbox"
+	maxHealth = 100
+	health = 100
+	threat_level = ZAYIN_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(30, 40, 50, 60, 70),
+		ABNORMALITY_WORK_INSIGHT = 60,
+		ABNORMALITY_WORK_ATTACHMENT = list(70, 60, 50, 40, 30),
+		ABNORMALITY_WORK_REPRESSION = 0,
+		)
+	pixel_x = 0
+	base_pixel_x = 0
+
+	work_damage_amount = 5
+	work_damage_type = RED_DAMAGE
+
+	gift_type =  /datum/ego_gifts/mail
+	gift_message = "An empty envelope slowly makes its way into your hands."
+
+	max_boxes = 12
+
+	ego_list = list(
+		/datum/ego_datum/weapon/letter_opener,
+		/datum/ego_datum/armor/letter_opener
+		)
+	abnormality_origin = ABNORMALITY_ORIGIN_ORIGINAL
+
+	var/cooldown
+	var/cooldown_time = 10 SECONDS
+	var/spawned_effects = list()
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/Destroy()
+	..()
+	for(var/obj/effect/VFX in spawned_effects)
+		VFX.Destroy()
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/Initialize(mapload)
+	. = ..()
+
+	for(var/i = 1 to 10)
+		var/turf/dispense_turf = get_step(src, i)
+		var/obj/effect/letters_flow/TMP = new(dispense_turf)
+		spawned_effects += TMP
+
+//papercut due to failed work
+/mob/living/simple_animal/hostile/abnormality/mailpile/WorktickFailure(mob/living/carbon/human/user)
+	if(prob(10))
+		to_chat(user,"<span class='warning'>Ouch! I got a paper cut!</span>")
+	return
+/mob/living/simple_animal/hostile/abnormality/mailpile/proc/Delivery(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(work_type == ABNORMALITY_WORK_REPRESSION)
```

Comment:
I will note this is very hard to do as right now you need like, 4+ boxes on a 0% work success.